client/flutter/lib/module_settings/ui/settings_page/settings_page.dart

```diff
@@ -28,145 +29,192 @@ class SettingsScreen extends StatefulWidget {
-                            Text(S.of(context).login),
-                            IconButton(
-                                icon: Icon(Icons.login),
-                                onPressed: () {
-                                  Navigator.of(context).pushNamed(
-                                      AuthorizationRoutes.LOGIN_SCREEN);
-                                })
-                          ],
-                        );
-                      }
+            Container(
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(10),
+                color: Theme.of(context).backgroundColor,
+              ),
+              child: Flex(
+                direction: Axis.vertical,
+                mainAxisAlignment: MainAxisAlignment.spaceAround,
+                children: [
+                  SizedBox(
+                    height: 16,
+                  ),
+                  ListTileSwitch(
+                    value: Theme.of(context).brightness == Brightness.dark,
+                    leading: Icon(
+                        Theme.of(context).brightness == Brightness.dark
+                            ? Icons.nightlight_round_rounded
+                            : Icons.wb_sunny),
+                    onChanged: (mode) {
+                      widget._themeDataService
+                          .switchDarkMode(mode)
+                          .then((value) {});
                     },
+                    visualDensity: VisualDensity.comfortable,
+                    switchType: SwitchType.cupertino,
+                    switchActiveColor: Theme.of(context).primaryColor,
+                    title: Text(S.of(context).darkMode),
                   ),
-                ),
+                  ListTile(
+                    leading: Icon(Icons.language),
+                    title: Text(S.of(context).language),
+                    trailing: DropdownButton(
+                        value: Localizations.localeOf(context).languageCode,
+                        underline: Container(),
+                        icon: Padding(
+                          padding: const EdgeInsets.all(4.0),
+                          child: Icon(Icons.arrow_drop_down_rounded),
+                        ),
+                        items: [
+                          DropdownMenuItem(
+                            child: Text('العربية'),
+                            value: 'ar',
+                          ),
+                          DropdownMenuItem(
+                            child: Text('English'),
+                            value: 'en',
+                          ),
+                        ],
+                        onChanged: (newLang) {
+                          widget._localizationService
+                              .setLanguage(newLang.toString());
+                        }),
+                  ),
+                  SizedBox(
+                    height: 16,
+                  ),
+                  // Flex(
```

Comment:
let us remove unused comment code

Reply:
Actually I just comment it for now I need it later for something else